From c1109b707b1eda5d98eb2ed6bd74083bc352f482 Mon Sep 17 00:00:00 2001
From: chrome-release-bot <chrome-release-bot@chromium.org>
Date: Fri, 6 Dec 2019 01:09:50 +0000
Subject: [PATCH] Incrementing VERSION to 78.0.3904.130

TBR=govind@chromium.org

Change-Id: Ib675653e2605cf2c15ebdbd1320465b08a49994d
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1954709
Reviewed-by: Chrome Release Bot (LUCI) <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/3904@{#917}
Cr-Branched-From: 675968a8c657a3bd9c1c2c20c5d2935577bbc5e6-refs/heads/master@{#693954}
---
 chrome/VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index 7a4383a1ccb7f..50f887f746227 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=78
 MINOR=0
 BUILD=3904
-PATCH=108
+PATCH=130
